service/src/main/resources/migrations/v019__backfila.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE backfill_runs
+    ADD COLUMN `deleted_at` TIMESTAMP NULL DEFAULT NULL;
```

Comment:
Follow the existing formatting
`  deleted_in_service_at timestamp(3) NULL DEFAULT NULL,`